docs/quick_modifiers.txt

```diff
@@ -10,6 +11,7 @@ quick_province_modifier = {
 }
 
 quick_country_modifier = {
+	id = <string> #optional
```

Comment:
why adding id-ies on quick modifiers?

Reply:
so we can also later remove them or add them again in another piece of code